client/src/helpers/APICalls/login.ts

```diff
@@ -1,6 +1,22 @@
-import { AuthApiData } from '../../interface/AuthApiData';
+import { AuthApiData, DemoAuthData } from '../../interface/AuthApiData';
 import { FetchOptions } from '../../interface/FetchOptions';
 
+export const demoLogin = async (email: string, password: string): Promise<DemoAuthData> => {
+  const fetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email, password }),
+  };
+
+  return await fetch(`${process.env.PUBLIC_URL}/auth/demo-login`, fetchOptions)
```

Comment:
@moffatethan  I didn't make the enviroment variable I realized this is what connect to the backend because I was called in other part of the app, bit strange, it worked from the box. I didn't set it myself

Reply:
Is the sample.env including this env var? If not, could you add it in. This isn't really about what you have done and didn't do, it's about keeping our codebase healthy and sometimes you might have to take matters into your own hands 😄